src/video/windows/SDL_windowsevents.c

```diff
@@ -1168,7 +1168,7 @@ WIN_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
         {
             SDL_assert(!g_ModalLoopWindow);
             g_ModalLoopWindow = data->window;
-            SetTimer(hwnd, (UINT_PTR)g_ModalLoopWindow, USER_TIMER_MINIMUM, NULL);
+            SetTimer(hwnd, (UINT_PTR)g_ModalLoopWindow, 0x0000000A, NULL);
```

Comment:
Instead of this, you can add the missing macro around line 98 or something among the others. E.g. like:
```c
#ifndef USER_TIMER_MINIMUM
#define USER_TIMER_MINIMUM 0x0000000A
#endif
```

Reply:
Thanks for the review, I applied your suggestions to the PR